easybuild/easyconfigs/l/libdap/libdap-3.20.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'libdap'
+version = '3.20.3'
+
+homepage = 'http://opendap.org/download/libdap'
+description = """A C++ SDK which contains an implementation of DAP 2.0
+ and the development versions of DAP3, up to 3.4.
+ This includes both Client- and Server-side support classes."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['http://www.opendap.org/pub/source/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['29961922b53f62e9d4eb34d1d50ddc23a24100664f97b71f42561fa5588ccc58']
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('Bison', '3.0.5'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('cURL', '7.60.0'),
+    ('libxml2', '2.9.8'),
+    ('LibUUID', '1.0.3'),
```

Comment:
Hmm, overlooked this, should be `util-linux` (cfr. #6963)

Fixed in #8258, @smoors please take a look?